src/app/workflow/dag/dag.component.html

```diff
@@ -32,6 +32,14 @@
             <input type="radio" name="dagType" [(ngModel)]="dagType" value="cwlviewer">Common Workflow Language Viewer
           </label>
         </div>
+        <div *ngIf="(isWDL$ | async) && (descriptorType$ | async) === ToolDescriptor.TypeEnum.WDL" style="line-height: 20px;">
```

Comment:
Isn't `(descriptorType$ | async) === ToolDescriptor.TypeEnum.WDL` redundant? It looks like that is what `isWDL$` is already checking.